pdcupdater/commands.py

```diff
@@ -35,7 +35,7 @@ def retry():
 def _initialize_basics(pdc):
     """ Gotta have these before we can really do anything... """
     arches = [dict(name=name) for name in ["armhfp", "i386", "x86_64"]]
-    pdc_arches = list(pdc_client.get_paged(pdc['arches']._))
+    pdc_arches = list(pdc.get_paged(pdc['arches']._))
```

Comment:
Ok, sounds safe then but I'll let @ralphbean do the merge since he knows the code and systems better than I :)